ocaml/idl/datamodel_host.ml

```diff
@@ -952,11 +952,33 @@ let host_query_ha = call ~flags:[`Session]
       ~pool_internal:true
       ~hide_from_docs:true
       ~name:"certificate_sync"
-      ~doc:"Resync installed SSL certificates and CRLs."
+      ~doc:"Make installed TLS CA certificates and CRLs available to all programs using OpenSSL."
       ~params:[Ref _host, "host", "The host"]
       ~allowed_roles:_R_LOCAL_ROOT_ONLY
       ()
 
+  let install_server_certificate = call
+      ~in_oss_since:None
+      ~lifecycle:[Published, rel_stockholm, ""]
+      ~name:"install_server_certificate"
+      ~doc:"Install the TLS server certificate."
+      ~versioned_params:
+        [{ param_type=Ref _host; param_name="host"; param_doc="The host"
+         ; param_release=stockholm_release; param_default=None}
+        ;{ param_type=String; param_name="certificate"
+         ; param_doc="The server certificate, in PEM form"
+         ; param_release=stockholm_release; param_default=None}
+        ;{ param_type=String; param_name="private_key"
+         ; param_doc="The unencrypted private key used to sign the certificate, \
+                      in PKCS#8 form"
+         ; param_release=stockholm_release; param_default=None}
+        ;{ param_type=String; param_name="certificate_chain"
```

Comment:
Chain to...? Any root CA installed system-wide, or the above installed TLS CA certificates with the other API, or both?

Reply:
None of those, it's to the first self-signed cert / root CA cert that the other party  in the TLS connection encounters in the verification chain.